en/components/general/wpf_to_angular_guide/angular_pipes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Angular Pipes
+_description: Learn how to transform data in Angular similar to converters in WPF.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, Pipe, Converter
+---
+
+## Transforming Data With Angular Pipes
+
+In WPF in order to transform data from one value to another you are going to use [IValueConverter](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.ivalueconverter?view=netframework-4.8). To transform data in an Angular application, we are going to use a [Pipe](https://angular.io/api/core/Pipe). The pipe is very similar to the WPF converter. It takes data as an input and then transforms that data into a desired output for display.
+
+Angular provides a number of predefined pipes for the most common data transformations. For example, if you want a header text to appear in upper case, you could use the [UpperCasePipe](https://angular.io/api/common/UpperCasePipe). Using a pipe is very simple. After your expression, provide a pipe operator `|` followed by the pipe name.
+
+```html
+<h2>{{ header | uppercase }}</h2>
+```
+
+What is really great about pipes in Angular compared to a WPF converter is that you can actually use more than one pipe at a time. So for example you can use a [DatePipe](https://angular.io/api/common/DatePipe) and [UpperCasePipe](https://angular.io/api/common/UpperCasePipe) in order to format the date to a user-friendly format and make it uppercase:
```

Comment:
- about **the** pipes in Angular compared to ...
- missing comma:  So, for example, you can use ...

Reply:
Addressed.